docs/samples/android.md

```diff
@@ -0,0 +1,51 @@
+## Using the Android sample bundle
+
+The Android example bundle in `samples/android` shows how to connect to an android device and turn on WLAN.
+
+1. clone [this](https://github.com/noeppi-noeppi/nodecg-io-android) Git Repository
+
+2. Run `gradlew build` (on windows) or `./gradlew build` (on linux) inside the cloned repository.
```

Comment:
`./gradlew build` throws lint errors for me:
```
> Task :app:lint                                                                                                                                                                                                   
Ran lint on variant release: 37 issues found                                                                                                                                                                       
Ran lint on variant debug: 38 issues found                                                                                                                                                                         
Wrote HTML report to file:///home/daniel/Coding/noeppi-noeppi/nodecg-io-android/app/build/reports/lint-results.html                                                                                                
Wrote XML report to file:///home/daniel/Coding/noeppi-noeppi/nodecg-io-android/app/build/reports/lint-results.xml                                                                                                  

> Task :app:lint FAILED                                                                                  

FAILURE: Build failed with an exception.                                                                 

* What went wrong:                                  
Execution failed for task ':app:lint'.                                                                   
> Lint found errors in the project; aborting build.                                                      
                                                                                                         
  Fix the issues identified by lint, or add the following to your build script to proceed with errors:                                                                                                             
  ...                                               
  android {                                         
      lintOptions {                                 
          abortOnError false                        
      }                                             
  }                                                 
  ...                                               
                                                    
  The first 3 errors (out of 11) were:                                                                   
  /home/daniel/Coding/noeppi-noeppi/nodecg-io-android/app/src/main/java/io/github/noeppi_noeppi/nodecg_io_android/Actions.java:247: Error: Call requires permission which may be rejected by user: code should explicitly check to see if permission is available (with checkPermission) or explicitly handle a potential SecurityException [MissingPermission]
          Location lastKnown = mgr.getLastKnownLocation(LocationManager.GPS_PROVIDER);                                                                                                                             
                               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                                                                                                                              
  /home/daniel/Coding/noeppi-noeppi/nodecg-io-android/app/src/main/java/io/github/noeppi_noeppi/nodecg_io_android/Actions.java:291: Error: Call requires permission which may be rejected by user: code should explicitly check to see if permission is available (with checkPermission) or explicitly handle a potential SecurityException [MissingPermission]
          mgr.requestLocationUpdates(LocationManager.GPS_PROVIDER, time, distance, listener);                                                                                                                      
          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                                                                                                                       
  /home/daniel/Coding/noeppi-noeppi/nodecg-io-android/app/src/main/java/io/github/noeppi_noeppi/nodecg_io_android/Actions.java:420: Error: Call requires permission which may be rejected by user: code should explicitly check to see if permission is available (with checkPermission) or explicitly handle a potential SecurityException [MissingPermission]
          String number = mgr.getLine1Number();                                                          
                          ~~~~~~~~~~~~~~~~~~~~                                                           
                                                    


* Try:                                              
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org                                                               

BUILD FAILED in 24s
```
So you it should say `./gradlew assemble` here (which worked for me) or you should fix the lint errors.

Reply:
lint should be fixed now